awx/ui_next/src/screens/Credential/CredentialDetail/CredentialDetail.jsx

```diff
@@ -75,34 +97,84 @@ function CredentialDetail({ i18n, credential }) {
 
   const { error, dismissError } = useDismissableError(deleteError);
 
-  const renderDetail = ({ id, label, type }) => {
-    let detail;
+  const renderDetail = ({ id, label, type, ask_at_runtime }) => {
```

Comment:
No need to nest this function definition inside of the `CredentialDetail`, it can be moved up and out.